### PR TITLE
Release version 3.8 (#207)

### PR DESCRIPTION
**Pull request by whimboo**
This PR is the fix for issue #207 and adds the history changes and bumps the version number.

@xabolcs can you please review? Thanks

### CODE REVIEW

**Comment by UtiluMark** [comment]
Shouldn't Seamonkey's maxVersion be 2.48 instead of 2.40?

**Comment by UtiluMark** [comment]
Notes on the changelog:
- In previous versions each line was indented with two spaces "  ", but the 3.8 changes are indented with only one space " ".
- In previous versions each line was prefixed with either "Repository: " or "Extension: ", but the 3.8 changes don't have prefixes.
- "Release version 3.8 (#207)" isn't worth mentioning in the history of changes I think, none earlier version has a similar release line.
- "Update minVersions for applications (#207)" would be worth mentioning.

**Comment by tonymec** [comment]
FWIW:
* 2.40 is the current stable version of SeaMonkey, which is getting a little old in the teeth.
* The developers hope to be able to release SeaMonkey 2.45 "soon" but there is no firm date yet. Pre-release builds are already available in subdirectories of http://ftp.mozilla.org/pub/seamonkey/tinderbox-builds/ with subdirectory names starting "comm-release-" followed by "linux" "linux64" "macosx64" or "trunk-win32" depending on platform. After selecting the appropriate page for your platform, scroll to the bottom of the page, then select the _numbered_ subfolder with the highest number.
-- SeaMonkey 2.45 will include Lightning and Devtools; its pre-releases already do.
* 2.46 is the current beta version.
* 2.47a2 is the current Aurora version.
* 2.48a1 is the current Trunk version.

File: History.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+3.8 / 2016-08-18
+==================
+
+ * Release version 3.8 (#207)
```
Comment (xabolcs) [inline History.md]: No need for this line, IMHO.

File: History.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+ * Remove reference to Screengrab extension (#188)
+ * Reduce top margin of main heading in about:nightly (#188)
+ * Update maxVersions for applications (#188)
+ * Fix white-space issues (#188)
+ * Removal of unused files (#188)
+ * Sort variables alphabetically in customize.js (#172)
+ * Add .editorconfig file (#174)
+ * Add Russian locale (#175)
+ * Fix retrieval of compiler name in case of extra dashes (#177)
+ * Add new variable for visible tabs (#173)
```
Comment (xabolcs) [inline History.md]: As @UtiluMark pointed out, one more space before `*`, for all list items above, please!
If you generated this with a tool then I hope that the indent is configurable.

File: History.md (hunk 1 above)
Comment (xabolcs) [inline History.md]: Or think about what @UtiluMark [suggested][2].


[2]: ../208#issuecomment-240763150

File: extension/install.rdf [hunk 3]
```diff
@@ -47,35 +47,26 @@
...
-    </em:targetApplication>
-
-    <!-- Songbird version -->
-    <em:targetApplication>
-      <Description>
-        <em:id>songbird@songbirdnest.com</em:id>
-        <em:minVersion>1.6</em:minVersion>
-        <em:maxVersion>2.2.0</em:maxVersion>
+        <em:minVersion>2.38</em:minVersion>
+        <em:maxVersion>2.40</em:maxVersion>
```
Comment (xabolcs) [inline extension/install.rdf]: SeaMonkey's `maxVersion` should be `2.48` as @UtiluMark and @tonymec suggested.
[Application versions][2] should taken from AMO.

[2]: https://addons.mozilla.org/en-US/firefox/pages/appversions/

File: History.md (hunk 2 above)
Comment (xabolcs) [inline History.md]: Also for prefixes! Thanks!

**Comment by whimboo** [comment]
> In previous versions each line was indented with two spaces "  ", but the 3.8 changes are indented with only one space " ".

We should better correct the older entries as continuing with two spaces.

> In previous versions each line was prefixed with either "Repository: " or "Extension: ", but the 3.8 changes don't have prefixes.

I feel this is obsolete information given that most changes will affect the extension. We could save us from this hand-crufting additions for each version by specifying explicitly if a change was done to the repository, and remove all 'Extension' prefixes. Does that sound ok?

> "Release version 3.8 (#207)" isn't worth mentioning in the history of changes I think, none earlier version has a similar release line.
> "Update minVersions for applications (#207)" would be worth mentioning.

Ok, I will update this line so we mention the min version bump. 

Also thanks for pointing out the current to hopefully be released version of SeaMonkey.

I will update the PR once we reached consensus about the prefixes.

**Comment by xabolcs** [comment]
> We should better correct the older entries as continuing with two spaces.

I'm OK with changing the old entries.

> I feel this is obsolete information given that most changes will affect the extension. We could save us from this hand-crufting additions for each version by specifying explicitly if a change was done to the repository, and remove all 'Extension' prefixes. Does that sound ok?

It's OK to me to remove all the `Extension:` prefixes.

**Comment by whimboo** [comment]
I pushed an update for this PR. So please check again. Thanks.

File: History.md [hunk 4]
```diff
@@ -1,63 +1,83 @@
...
+ * Vendor of Thunderbird should be "Mozilla" instead of being empty (#188)
+ * Add pushlog feature for all branches (#51)
+ * Remove non-working image host allyoucanupload.webshots.com (#188)
+ * Remove reference to Screengrab extension (#188)
+ * Reduce top margin of main heading in about:nightly (#188)
+ * Update maxVersions for applications (#188)
+ * Fix white-space issues (#188)
+ * Removal of unused files (#188)
+ * Sort variables alphabetically in customize.js (#172)
+ * Add .editorconfig file (#174)
```
Comment (xabolcs) [inline History.md]: This needs a `Repository:` prefix, IMHO.

**Comment by xabolcs** [comment]
Thanks!
Looks good, only one note about `.editorconfig`. See inline comment!

File: History.md (hunk 4 above)
Comment (whimboo) [inline History.md]: It wasn't marked as such. I can add it.

**Comment by whimboo** [comment]
An update has been pushed. Will land the patch now.

**Pull request merged**